clang/lib/Frontend/Rewrite/FrontendActions.cpp

```diff
@@ -185,7 +185,7 @@ RewriteObjCAction::CreateASTConsumer(CompilerInstance &CI, StringRef InFile) {
 void RewriteMacrosAction::ExecuteAction() {
   CompilerInstance &CI = getCompilerInstance();
   std::unique_ptr<raw_ostream> OS =
-      CI.createDefaultOutputFile(/*Binary=*/true, getCurrentFileOrBufferName());
+      CI.createDefaultOutputFile(/*Binary=*/false, getCurrentFileOrBufferName());
```

Comment:
I don't understand why this would need to be changed. I don't think the output backend changes anything here.

Reply:
(i.e., if it was previously "binary", then it should basically hit the same code paths in OnDiskOutputBackend as it used to hit in CompilerInstance — any fixes should either be in the call to `createFile` or in the outputbackend itself)